components/circle/sources/new-comment-posted/new-comment-posted.mjs

```diff
@@ -0,0 +1,94 @@
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+import circle from "../../circle.app.mjs";
+import sampleEmit from "./test-event.mjs";
+
+export default {
+  key: "circle-new-comment-posted",
+  name: "New Comment Posted",
+  description: "Emit new event each time a new comment is posted in the selected community space.",
+  version: "0.0.1",
+  type: "source",
+  dedupe: "unique",
+  props: {
+    circle,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: DEFAULT_POLLING_SOURCE_TIMER_INTERVAL,
+      },
+    },
+    communityId: {
+      propDefinition: [
+        circle,
+        "communityId",
+      ],
+    },
+    spaceId: {
+      propDefinition: [
+        circle,
+        "spaceId",
+        ({ communityId }) => ({
+          communityId,
+        }),
+      ],
+      optional: true,
+    },
```

Comment:
The props `circle`, `db`, `timer`, `communityId`, and `spaceId` could all be put in a common file.